shared-bindings/protomatter/Protomatter.c

```diff
@@ -314,10 +314,106 @@ STATIC mp_obj_t protomatter_protomatter_refresh(mp_obj_t self_in) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(protomatter_protomatter_refresh_obj, protomatter_protomatter_refresh);
 
+//|   .. attribute:: width
+//|
+//|     The width of the display, in pixels
+//|
+STATIC mp_obj_t protomatter_protomatter_get_width(mp_obj_t self_in) {
+    protomatter_protomatter_obj_t *self = (protomatter_protomatter_obj_t*)self_in;
+    check_for_deinit(self);
+    return MP_OBJ_NEW_SMALL_INT(common_hal_protomatter_protomatter_get_width(self));
+}
+MP_DEFINE_CONST_FUN_OBJ_1(protomatter_protomatter_get_width_obj, protomatter_protomatter_get_width);
+const mp_obj_property_t protomatter_protomatter_width_obj = {
+    .base.type = &mp_type_property,
+    .proxy = {(mp_obj_t)&protomatter_protomatter_get_width_obj,
+              (mp_obj_t)&mp_const_none_obj,
+              (mp_obj_t)&mp_const_none_obj},
+};
+
+//|   .. attribute:: height
+//|
+//|     The height of the display, in pixels
+//|
+STATIC mp_obj_t protomatter_protomatter_get_height(mp_obj_t self_in) {
+    protomatter_protomatter_obj_t *self = (protomatter_protomatter_obj_t*)self_in;
+    check_for_deinit(self);
+    return MP_OBJ_NEW_SMALL_INT(common_hal_protomatter_protomatter_get_height(self));
+}
+MP_DEFINE_CONST_FUN_OBJ_1(protomatter_protomatter_get_height_obj, protomatter_protomatter_get_height);
+
+const mp_obj_property_t protomatter_protomatter_height_obj = {
+    .base.type = &mp_type_property,
+    .proxy = {(mp_obj_t)&protomatter_protomatter_get_height_obj,
+              (mp_obj_t)&mp_const_none_obj,
+              (mp_obj_t)&mp_const_none_obj},
+};
+
+//|   .. attribute:: bytes_per_cell
+//|
+//|     The bytes_per_cell of the display, in pixels.  Always equal to 1.
```

Comment:
Why have three attributes that are always the same value?

Reply:
So that they can be queried from the object properties by a FramebufferDisplay.

Reply:
other framebuffers in the future will presumably have different values in these properties

Reply:
Do we expect Python-level framebuffer objects? I was thinking this could be fetched through the underlying protocol instead of the python api.

Reply:
No, but python code that writes a framebuffer directly without FramebufferDisplay would need the values too. I can add it to the protocol, or move it if you feel strongly

Reply:
I went ahead and removed the properties but we can revert that commit if it becomes useful in the future.